snooty/postprocess.py

```diff
@@ -301,6 +303,66 @@ def populate(self, fileid_stack: FileIdStack, node: n.Node) -> None:
         node.refuri = refuri
 
 
+class ContentsHandler:
```

Comment:
A docstring would be nice to explain what this is looking for and what it does to the AST